### ISSUE TITLE
Fix S2275: Rule should not report format string error on certain method...

### ISSUE THREAD

**Issue by Evangelink** [SonarSource/sonar-csharp#258]
...when no parameters are provided.

Among the listed methods, the one not ending with `Format` allows to use pattern considered as invalid when no parameters are used (in this case the `string` overload is used).
For example, the following pattern is allowed for `Console.WriteLine` but not for `string.Format`:

```
Console.WriteLine("{");
````
